wolox-books-angularjs/src/app/config/development.js

```diff
@@ -1,3 +1,3 @@
 angular.module('app').constant('configuration', {
-  apiUrl: 'localhost:3000'
+  apiUrl: 'https://wbooks-api-stage.herokuapp.com/api/v1'
```

Comment:
use a .env file

Reply:
This file structure matches the one defined in Frontend-Bootstrap repo